tests/full-charmcraft.yaml/go.mod

```diff
@@ -0,0 +1,13 @@
+module go-app
+
+go 1.22.6
```

Comment:
Is it on purpose that the test app requires a different go version that the generator ?